client/blocks/inline-help/style.scss

```diff
@@ -380,7 +379,8 @@
 
 		a {
 			color: var( --color-text-inverted );
-			&:hover {
+			&:hover,
+			&:focus {
```

Comment:
Just checking why this got added. The original omission seems fairly intentional. Does it fix a bug?

Reply:
It was added to simplify the styles applied in the help-search element. I don't see any big deal adding this pseudo-class here in order to reduce (and improve) the code.
Even more, I think we should move all `focus` styles to the inline-help but in another PR. The changes are increasing too much 😅

Reply:
I don't think it does anything. There is no background on these `a` elements. 

I _think_ the intention was to provide a indicator to the user when the hovered or focused a "selected" result item in order that it was obvious where the current focus was.

Perhaps what we want is:

```scss
&.is-selected {
    background: var( --color-primary );
    cursor: pointer;
    color: var( --color-text-inverted );

    &:hover,
    &:focus {
        background: none;
    }

    a {
        color: var( --color-text-inverted );
    }
}
```

Notice how I've moved the focus styles onto the `<li>` so that we can see when the selected item is focused. Try this out by using arrows to select an item and then using `TAB` to move the focus around.

Reply:
yup, I think it deserves an :octocat: issue too.

Reply:
I see you [already create it](https://github.com/Automattic/wp-calypso/issues/43615) 👏 We can keep improving the approach there.